src/httpserver/file_io/file_input.clj

```diff
@@ -0,0 +1,17 @@
+(ns httpserver.file-io.file-input
+  (:require [clojure.java.io :as io])
+  (:import java.io.FileInputStream))
+
+(defn- partial-content [array-of-bytes start end]
+  (subvec (mapv byte array-of-bytes) start end))
+
```

Comment:
It applies to the function above `partial-content` and the method `get-file-partial -content`.  It does not apply to the function `get-file-contents-as-bytes`